DOC redirect to GitHub issues & PR instead of mailing list when it comes to contribution (#30177)
scikit-learn/scikit-learn: doc/developers/contributing.rst

12. Follow `these
    <https://help.github.com/articles/creating-a-pull-request-from-a-fork>`_
    instructions to create a pull request from your fork. This will send an

>     email to the committers. You may want to consider sending an message to

Reviewer: ```suggestion
    notification to potential reviewers. You may want to consider sending an message to
```

    the `discord <https://discord.com/invite/h9qyrK8Jc8>`_ in the development

>     chanel for more visibility (instant replies are not guaranteed).

Reviewer 2: ```suggestion
    channel for more visibility (instant replies are not guaranteed).
```

>     the `discord <https://discord.com/invite/h9qyrK8Jc8>`_ in the development
>     channel for more visibility (instant replies are not guaranteed).

Reviewer: ```suggestion
    the `discord <https://discord.com/invite/h9qyrK8Jc8>`_ in the development
    channel for more visibility if your pull request does not receive attention after
    a couple of days (instant replies are not guaranteed though).
```
